src/JitInterface/src/CorInfoTypes.cs

```diff
@@ -1241,7 +1253,12 @@ public struct VarLoc
        public int vlType;
        // The 64bit field is here to keep VarLoc 8byte aligned on Amd64.
        // For x86, we need to change the VarLoc definition here.
-       public long A;   
+#if BIT64
```

Comment:
@jkotas It's strange for me but with this variant I've got the following during build (both on x64 and armel):
```
/share/CORERT/corert/src/JitInterface/src/CorInfoTypes.cs(1255,13): error CS0169: The field 'VarLoc.C' is never used [/share/CORERT/corert/src/System.Private.Jit/src/System.Private.Jit.csproj]
/share/CORERT/corert/src/JitInterface/src/CorInfoTypes.cs(1256,13): error CS0169: The field 'VarLoc.D' is never used [/share/CORERT/corert/src/System.Private.Jit/src/System.Private.Jit.csproj]
/share/CORERT/corert/src/JitInterface/src/CorInfoTypes.cs(1253,16): error CS0169: The field 'VarLoc.A' is never used [/share/CORERT/corert/src/System.Private.Jit/src/System.Private.Jit.csproj]
/share/CORERT/corert/src/JitInterface/src/CorInfoTypes.cs(1254,13): error CS0169: The field 'VarLoc.B' is never used [/share/CORERT/corert/src/System.Private.Jit/src/System.Private.Jit.csproj]
```